public/app/plugins/datasource/loki/types.ts

```diff
@@ -28,6 +28,8 @@ export enum LokiQueryType {
   // Stream = 'stream',
 }
 
+export type LokiDirectionType = 'FORWARD' | 'BACKWARD';
```

Comment:
Do you think it would make sense to change this to enum as `LokiQueryType` is also enum?

Reply:
Yes this makes things more consistent.